readthedocs/oauth/models.py

```diff
@@ -62,6 +62,10 @@ def is_admin(self):
             return full_json['permissions']['admin']
         return False
 
+    def is_private(self):
+        full_json = eval(self.json)
```

Comment:
Oops, commented in the commit directly which doesn't show up here: https://github.com/rtfd/readthedocs.org/commit/2f3d32162d130a3c4b0fe430a447164ddce15f9d#commitcomment-12682730

Please don't use eval. It's a security problem and it's not the right parsing algorithm. JSON looks a bit like Python dicts and lists. But It isn't! One example:

```
>>> import json
>>> json.loads('{"hello": true}')
{u'hello': True}
>>> eval('{"hello": true}')
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "<string>", line 1, in <module>
NameError: name 'true' is not defined
```

Also unicode handling is different:
```
>>> json.dumps({'Hällo': 'Straßenbahn'})
'{"H\\u00e4llo": "Stra\\u00dfenbahn"}'
>>> json_string = json.dumps({'Hällo': 'Straßenbahn'})
>>> print json.loads(json_string)[u'Hällo']
Straßenbahn
>>> print eval(json_string)[u'Hällo']
Traceback (most recent call last):
  File "<input>", line 1, in <module>
KeyError: u'H\xe4llo'
```